### ISSUE TITLE
When a player falls with his bike directly into the water, he dies.

### ISSUE THREAD

**Issue by codectile** [oscar-broman/samp-weapon-config#17]
I was just testing this include and I came to know that as soon as a player falls with his bike into the water, he dies immediately , and on the screen it displays "collision -100.0". It is causing a major problem to the "Fall from moving vehicle damage system". Without it being fixed, I cannot proceed further.

https://www.youtube.com/watch?v=gc1-BS39G6M

**Comment by Crayder** [comment]
Well it would be your "Fall from moving vehicle damage system" that is causing it right? If so it is your problem. Maybe check to see if the vehicle was a bike vehicle. Also, I think OnVehicleDeath is called when bike is entering water. Maybe you could add a hook for that to check.

**Comment by codectile** [comment]
No, if it was the falt of "fall from moving vehicle" I would not create an issue here.

**Comment by Crayder** [comment]
It doesn't give me damage, tested by falling into the water from 100 meters above sea level on an nrg. The only thing called was OPTakeD: 
![image](https://cloud.githubusercontent.com/assets/9919010/6312880/9aa1e116-b959-11e4-9290-73d49dd32d7b.png)
How do I recreate your scenario?

**Comment by codectile** [comment]
Oh I am very sorry, by mistake I did download my fork again instead of the real version and I was thinking that it's the real version. Sorry once again.

**Comment by Crayder** [comment]
We all make mistakes, it's ight, no harm done. We could still consider this though:

OnPlayerTakeDamage, detects bodypart 3, reason 54, invalid player id, and exactly 0.0 health when falling into the water on a bike. We could add an amount of damage to that from the lastzvel for vehicle impact on water damage.

**Comment by codectile** [comment]
I think, I need to reopen this issue as I right, I guess. Crayder, I am going to send you a file in forums with a description go through it once.

**Comment by codectile** [comment]
Added video.
